backend/app/models/users.js

```diff
@@ -99,7 +101,8 @@ User.validPassword = function (pas, salt, checkpas) {
 User.editCols = [
     'firstName', 'lastName', 'phone', 'birthday', 'password',
     'updated', 'timezone', 'location', 'cell', 'address',
-    'lang', 'bio', 'notifyLevel', 'affiliation', 'isAnonymous', 'isDeleted',
+    'lang', 'bio', 'notifyLevel', 'affiliation', 'isAnonymous',
+    'authId', 'isDeleted',
```

Comment:
Should we let the `authId` change if we are going to have it? Shouldn't we set it on create and never let it change?

Reply:
For production, correct. For staging and testing, this can cause issues when we wipe out and refresh the database. But you're right, a better way of doing this would probably be to check the auth service to get or create the user when they're created in Greyscale, _then_ populate. I already have a ticket, INBA-609, which I've earmarked to improve the function that calls this. We can modify that to reflect this better approach.